app/views/content/train-to-be-a-teacher/if-you-have-a-degree.md

```diff
@@ -0,0 +1,79 @@
+---
+title: If you have a degree
+related_content:
+    Get support training to teach if you're disabled: "/get-support-training-to-teach-if-you-are-disabled"
+    Improve your subject knowledge : "/improve-your-subject-knowledge"
+    Tips on applying for teacher training : "/tips-on-applying-for-teacher-training"
+extra_content:
+    - content/train-to-be-a-teacher/if-you-have-a-degree/events-near-you
+---
+
+If you have a degree, or you’re currently studying for one, there are different routes you can take to getting [qualified teacher status](/what-is-qts) (QTS). You need QTS to work in the majority of schools in England including state maintained primary, secondary and special schools.
+
+We also have information for people outside of the UK who want to get [international qualified teacher status](/international-qualified-teacher-status) (iQTS) or [come to teach in England](/come-to-england-to-teach-if-you-are-a-teacher-from-outside-the-uk).
```

Comment:
Might be worth checking with the international team what the best phrase is to use here for 'people outside of the UK' as it may be relevant for international students that are currently in the UK?

Reply:
Also - we also have a page for international students who are not qualified teachers (https://getintoteaching.education.gov.uk/train-to-teach-in-england-as-an-international-student) - need to link to this too?

Reply:
I've changed the link the train to teach international one as it makes more sense on this page and also we can just say international student.